### ISSUE TITLE
ILM: Add option to disable setting indices to read-only when force merge is executed 

### ISSUE THREAD

**Issue by jakommo** [elastic/elasticsearch#53289]
**Describe the feature**:
Today, when a [force merge action is executed via ILM](https://www.elastic.co/guide/en/elasticsearch/reference/7.6/_actions.html#ilm-forcemerge-action), it will automatically mark the index read only, to prevent the user from the implications that would come with still writing to this index. 
It totally makes sense to have this as default.

In some scenarios, this is causing issues and it would be nice to have a setting to disable this behavior, accepting the potential problems this could cause. 

One such scenario is with Logstash sending data to indices that got forced merged via ILM and are read-only now.
This can either be because a user is back filling data or because there is an issue with date parsing etc.
Currently Logstash will log a warning that the index is read only, but will retry forever (which seems like the correct behavior) and eventually fill up the entire pipeline and block Logstash from processing any events. 

I was talking to users about this and they would much prefer to accept the data into the force merged indices and live with performance implications that come from having the small segments next to the huge segment, than having their Logstash pipeline blocked by this. 

As for workarounds, I only see something like a cronjob that uses curator or a script to "un-block" the indices that got force merges by ILM already.

**Comment by dakrone** [comment]
We discussed this and decided to remove the auto read-only setting. A user can manually use the `readonly` action if they want to set the index as read only.

**Comment by joegallo** [comment]
Related to #43426, which looks very similar to this issue.

**Comment by jakommo** [comment]
Good catch @joegallo . I'm closing this a dup of https://github.com/elastic/elasticsearch/issues/43426
